pkg/scheduler/cache/cache.go

```diff
@@ -1536,9 +1537,11 @@ func (sc *SchedulerCache) UpdateJobStatus(job *schedulingapi.JobInfo, updatePG b
 		if err != nil {
 			return nil, err
 		}
+		sc.Mutex.Lock()
+		sc.Jobs[job.UID].PodGroup.GetAnnotations()[api.TopologyAllocateLCAHyperNode] = job.PodGroup.GetAnnotations()[api.TopologyAllocateLCAHyperNode]
```

Comment:
I need to update cache and apiserver in case of scheduler restarted, when l can find out the LCA hyperNode of the job.

Reply:
According to the discussion with @Monokaix @JesseStutler , the LCAHypernode of the job is recorded in the annotations of the podgroup in case of scheduler restarted when l can find it out. So I need to update cache and apiserver  also.

Reply:
According to the discussion with @Monokaix and @JesseStutler, the LCAHypernode of the job is recorded in the annotations of the podgroup in case the Scheduler restarts and when I can find it out. So I also need to update the cache and the apiserver.